shared_context/active_filters_shared.rb

```diff
@@ -0,0 +1,108 @@
+require 'rspec'
+require 'watir'
+require 'helper'
+
+shared_context 'Filter Categories' do |filter_categories_data|
+  let(:method) { @main.methods }
```

Comment:
subcategory_data['sweaters'] should be renamed to e.g. filters_subcategory_value because it does not applies only to sweater anymore, and also should be changed in regression_filter.rb